flake.nix

```diff
@@ -17,22 +17,29 @@
     cardano-playground.flake = false; # otherwise, +9k dependencies in flake.lock…
   };
 
-  outputs = inputs:
+  outputs = inputs: let
+    inherit (inputs.nixpkgs) lib;
+  in
     inputs.flake-parts.lib.mkFlake {inherit inputs;} ({config, ...}: {
       imports = [
         inputs.devshell.flakeModule
         inputs.treefmt-nix.flakeModule
       ];
 
       flake.internal =
-        inputs.nixpkgs.lib.genAttrs config.systems (
+        lib.genAttrs config.systems (
           targetSystem: import ./nix/internal/unix.nix {inherit inputs targetSystem;}
         )
-        // inputs.nixpkgs.lib.genAttrs ["x86_64-windows"] (
+        // lib.genAttrs ["x86_64-windows"] (
           targetSystem: import ./nix/internal/windows.nix {inherit inputs targetSystem;}
         );
 
-      systems = ["x86_64-linux" "aarch64-linux" "aarch64-darwin" "x86_64-darwin"];
+      systems = [
+        "x86_64-linux"
+        # "aarch64-linux"
```

Comment:
I disabled `aarch64-linux`, because we don’t have CI for it (neither in `blockfrost-platform`, or `testgen-hs`, or even `cardano-node`).